### PR TITLE
Small fix in R3BSofFragmentAnalysis.cxx

### PR DESCRIPTION
**Pull request by ryotani**

### CODE REVIEW

**Comment by ryotani** [comment]
Is this error message in the CI-CD process related to my commit?

-----
CMake Error at /cvmfs/fairsoft.gsi.de/ubuntu2204/fairsoft/nov22p1/lib/G4Root-6.1.1/G4RootTargets.cmake:55 (set_target_properties):
  The link interface of target "g4root" contains:

    Geant4::G4Tree

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

**Comment by jose-luis-rs** [comment]
I will take a look later

File: sofana/R3BSofFragmentAnalysis.cxx [hunk 1]
```diff
@@ -267,11 +267,22 @@ void R3BSofFragmentAnalysis::Exec(Option_t* option)
...
+    {
+        HitMusic[ihit] = (R3BMusicHitData*)fMusicHitDataCA->At(ihit);
+        if (!HitMusic[ihit])
+            continue;
+        // In case the MusicHitData container has several "realistic" values,
+        // it's not possible to distinguish which is the "correct" event. Thus skipping events having several hits.
+        if (TMath::Abs(MusicTheta) < 0.1)
+            return;
+        MusicTheta = HitMusic[ihit]->GetTheta();
+    }
```
Comment (jose-luis-rs) [inline sofana/R3BSofFragmentAnalysis.cxx:292]: I think that you should ask for this condition before starting with the next calculations:
if (nHitMwpc0 < 1 || nHitMwpc1 < 1)
        return;

Comment (ryotani) [reply sofana/R3BSofFragmentAnalysis.cxx:292]: Thanks, I amended the code accordingly.

**Pull request merged**